src/rust/ide/view/graph-editor/src/component/node.rs

```diff
@@ -405,7 +411,7 @@ impl NodeModel {
         display_object.add_child(&visualization);
         display_object.add_child(&input);
 
-        let error_visualization = builtin_visualization::Error::new(&scene);
+        let error_visualization = builtin_visualization::Error::new(scene.clone_ref());
```

Comment:
why do we need to provide scene clone to `Error`? Also, I think our APIs should always allow passing references to clone-refable things and just do clonesinside, not to force users of the API to manually create clone refs on every call side (it is just API simpler to use).